contracts/interfaces/IScaledBalanceToken.sol

```diff
@@ -27,8 +27,8 @@ interface IScaledBalanceToken {
    * @dev Emitted after scaled balance tokens are burned
    * @param from The address from which the scaled tokens will be burned
    * @param target The address that will receive the underlying, if any
```

Comment:
The `VariableDebToken` is always passing `ZERO_ADDRESS` as target. I think its fine to do that as a simplification, but the real recipient of the underlying is the `AToken` contract...